apps/web/src/hooks/useMetadata.ts

```diff
@@ -1,46 +1,54 @@
 'use client'
 
-import { useEffect, useRef, useState } from 'react'
+import { useEffect, useState } from 'react'
 
 import type { ConversationMetadata } from 'promptl-ai'
 import { useDebouncedCallback } from 'use-debounce'
-
+import { APP_DOMAIN } from '$/app/(private)/_lib/constants'
 import type { ReadMetadataWorkerProps } from '../workers/readMetadata'
 
+const MANIFEST_PROMISE = fetch(`${APP_DOMAIN}/workers/worker-manifest.json`)
+  .then((res) => res.json())
+  .then((manifest) => `${APP_DOMAIN}${manifest['readMetadata.js']}`)
```

Comment:
ditto absolute path `/${manifest['readMetadata.js']}`

Reply:
I'm not sure it works for initializing the worker. I'll try and report back